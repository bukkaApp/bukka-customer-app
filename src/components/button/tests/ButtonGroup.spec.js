import React from 'react';

import ButtonGroup from '../ButtonGroup';

describe('Button Group component', () => {
  const wrapper = shallow(
    <ButtonGroup classNames="button-text">
      <button>CHILD</button>
    </ButtonGroup>
  );

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
