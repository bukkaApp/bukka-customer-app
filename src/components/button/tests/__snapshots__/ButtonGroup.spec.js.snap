// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Group component renders properly 1`] = `ShallowWrapper {}`;
